frontend/public/mockServiceWorker.js

```diff
@@ -0,0 +1,338 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (0.35.0).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
```

Comment:
so I ran `npm run build` and saw that `mockServiceWorker.js` is included in `frontend/build` (which presumably will change to `dist/frontend` once we configure it correctly). if we use `app.use(express.static(path.resolve('dist/frontend')))` (as per #2675) will that mean someone would be able to get this file from `form.gov.sg/mockServiceWorker.js`?

Reply:
Yes they would, but it would be useless because the service worker would not be registered anyways. I guess we can add a .dockerignore entry and it'll not be in the image

Reply:
sounds good